webapp/src/monaco.tsx

```diff
@@ -903,6 +917,18 @@ export class Editor extends toolboxeditor.ToolboxEditor {
         this.editor.setValue(content);
     }
 
+    private shouldShowToolbox(): boolean {
+        const readOnly =
+            !this.currFile
+            || this.currFile.isReadonly()
+            || pxt.shell.isReadOnly()
+            || this.isDebugging();
+        return pxt.appTarget.appTheme.monacoToolbox
+            && !readOnly
+            && ((this.fileType == "typescript" && this.currFile.name == "main.ts")
```

Comment:
Is there a reason to only show for `main.[ts|py]`? That's the current behavior so no reason to hold up this PR on it, but when I'm using the github integration to make a game sometimes I wish I had the toolbox to look up the js equivalent for a block I rarely use / not just have to guess it with intellisense - might be nice to have it show whenever it's editable and js/py

Reply:
Once you are beyond the default main.ts/.py, you are advanced enough that the toolbox is just annoying and eating up space. Intellisense is more than enough.

Reply:
Can be negotiated :)